packages/matchbox/src/components/Drawer/Header.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Close } from '@sparkpost/matchbox-icons';
+import { tokens } from '@sparkpost/design-tokens';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { Button } from '../Button';
+
+const Header = React.forwardRef(function Header(props, ref) {
+  const { children, onClose, showCloseButton } = props;
+  return (
+    <Box data-id="drawer-header" bg="white" p="500" ref={ref}>
+      <Box display="flex">
+        <Box flex="1">
+          <Text as="h1" fontSize="400" lineHeight="400">
```

Comment:
This should probably be an `h2` - only one `h1` on a page after all, and that should belong to the page heading.